lib/graphql/static_validation/all_rules.rb

```diff
@@ -1,23 +0,0 @@
-# Default rules for {GraphQL::StaticValidation::Validator}
-#
-# Order is important here. Some validators return {GraphQL::Language::Visitor::SKIP}
-# which stops the visit on that node. That way it doesn't try to find fields on types that
-# don't exist, etc.
-GraphQL::StaticValidation::ALL_RULES = [
```

Comment:
Sorry, maybe this is a bit out of the scope fo the refactor. 
The change is because it's kind of general rule to have files by class/module definitions. Also is adopted as a rule [here](https://github.com/bbatsov/ruby-style-guide#one-class-per-file). 
So, as it's a constant in `StaticValidation` module I thought that it was more consistent having it defined there.

I can revert this change if you prefer...